contrib/pkg/adm/managedns/enable.go

```diff
@@ -381,12 +343,9 @@ func (o *Options) getResourceHelper() (*resource.Helper, error) {
 
 func (o *Options) setupLocalClients() error {
 	log.Debug("creating cluster client config")
-	rules := clientcmd.NewDefaultClientConfigLoadingRules()
-	kubeconfig := clientcmd.NewNonInteractiveDeferredLoadingClientConfig(rules, &clientcmd.ConfigOverrides{})
-	cfg, err := kubeconfig.ClientConfig()
+	cfg, err := hiveutils.GetClientConfig()
 	if err != nil {
 		log.WithError(err).Error("cannot obtain client config")
-		return err
```

Comment:
Was the remove to the return intentional?

Reply:
absolutely not. good catch.